src/libraries/Microsoft.Extensions.DependencyInjection.Abstractions/src/ServiceDescriptor.cs

```diff
@@ -72,15 +72,30 @@ private ServiceDescriptor(Type serviceType, ServiceLifetime lifetime)
             ServiceType = serviceType;
         }
 
+        /// <summary>
+        /// The <see cref="ServiceLifetime"/> of the service.
```

Comment:
```suggestion
        /// Gets the <see cref="ServiceLifetime"/> of the service.
```

Reply:
Ideally this would provide info other than just the type, which is provided automatically anyway in both IntelliSense and docs.

Reply:
Normally I 100% agree with this, but my understanding with this class's functionality is that it keeps service information in the same place and the properties getting summaries in this PR are not changed aside from construction. This led me to believe that IntelliSense/doc type info already probably sufficed but for the sake of completeness I adapted the parameter descriptions on the constructors that eventually assign to these properties.